providers/os/resources/packages/windows_packages.go

```diff
@@ -245,19 +329,22 @@ func (w *WinPkgManager) List() ([]Package, error) {
 		pkgs = append(pkgs, appxPkgs...)
 	}
 
-	// hotfixes
-	cmd, err = w.conn.RunCommand(powershell.Wrap(WINDOWS_QUERY_HOTFIXES))
-	if err != nil {
-		return nil, errors.Wrap(err, "could not fetch hotfixes")
-	}
-	hotfixes, err := ParseWindowsHotfixes(cmd.Stdout)
-	if err != nil {
-		return nil, errors.Wrapf(err, "could not parse hotfix results")
-	}
-	hotfixAsPkgs := HotFixesToPackages(hotfixes)
-
-	pkgs = append(pkgs, hotfixAsPkgs...)
+	if !canRunCmd {
+		log.Debug().Msg("cannot run command on windows, skipping hotfix list")
+	} else {
```

Comment:
nit; if you return in the above `if`, you can avoid this `else`

(I think you know I like to reduce indentation to the limits 😂, up to you to agree or disagree 😅  )

Reply:
yep, actually i need to only check this once and return if false